qiskit/providers/backend.py

```diff
@@ -289,13 +80,6 @@ class BackendV2(Backend, ABC):
     something like a ``shots`` field for a backend that runs experiments which
     would contain an int for how many shots to execute.
 
-    If migrating a provider from :class:`~qiskit.providers.BackendV1`
-    one thing to keep in mind is for
-    backwards compatibility you might need to add a configuration method that
-    will build a :class:`~qiskit.providers.models.BackendConfiguration` object
-    and :class:`~qiskit.providers.models.BackendProperties` from the attributes
-    defined in this class for backwards compatibility.
```

Comment:
We could have left this comment in the docstring because someone could conceivably be migrating from `BackendV1` post removal (as part of fixing 2.0 compatibility). But it also is fine to remove it since this lives in the 1.4 docs.